Src/java/quick/src/main/resources/org/hl7/fhir/fhir-modelinfo-4.0.0.xml

```diff
@@ -0,0 +1,11684 @@
+                <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="FHIR" name="UsageContext"/>
+                <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="FHIR" name="Dosage"/>
+            </ns4:elementTypeSpecifier>
+        </ns4:element>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="CoverageEligibilityResponse.Error" retrievable="false" baseType="FHIR.BackboneElement">
+        <ns4:element name="code" elementType="FHIR.CodeableConcept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="CarePlanActivityKind" retrievable="false" baseType="FHIR.Element">
+        <ns4:element name="value" elementType="System.String"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="Composition.Attester" retrievable="false" baseType="FHIR.BackboneElement">
+        <ns4:element name="mode" elementType="FHIR.CompositionAttestationMode"/>
+        <ns4:element name="time" elementType="FHIR.dateTime"/>
+        <ns4:element name="party" elementType="FHIR.Reference"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="StructureMapSourceListMode" retrievable="false" baseType="FHIR.Element">
+        <ns4:element name="value" elementType="System.String"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="RequestStatus" retrievable="false" baseType="FHIR.Element">
+        <ns4:element name="value" elementType="System.String"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="OperationOutcome.Issue" retrievable="false" baseType="FHIR.BackboneElement">
+        <ns4:element name="severity" elementType="FHIR.IssueSeverity"/>
+        <ns4:element name="code" elementType="FHIR.IssueType"/>
+        <ns4:element name="details" elementType="FHIR.CodeableConcept"/>
+        <ns4:element name="diagnostics" elementType="FHIR.string"/>
+        <ns4:element name="location">
+            <ns4:elementTypeSpecifier xsi:type="ns4:ListTypeSpecifier" elementType="FHIR.string"/>
+        </ns4:element>
+        <ns4:element name="expression">
+            <ns4:elementTypeSpecifier xsi:type="ns4:ListTypeSpecifier" elementType="FHIR.string"/>
+        </ns4:element>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="strandType" retrievable="false" baseType="FHIR.Element">
+        <ns4:element name="value" elementType="System.String"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QuestionnaireResponseStatus" retrievable="false" baseType="FHIR.Element">
+        <ns4:element name="value" elementType="System.String"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="SubstancePolymer.Repeat" retrievable="false" baseType="FHIR.BackboneElement">
+        <ns4:element name="numberOfUnits" elementType="FHIR.integer"/>
+        <ns4:element name="averageMolecularFormula" elementType="FHIR.string"/>
+        <ns4:element name="repeatUnitAm
```

Comment:
I think this is intended for "SubstancePolymer" and to use the "class" element? That's how I applied it.

Reply:
TBH, I'm not sure what it was supposed to be applied to since GitHub screwed this all up (as you discovered later).  I'll review your last commit and ensure we're on the same page.